_observing-your-data/alerting/api.md

```diff
@@ -1350,15 +1334,16 @@ The following table lists the available path parameters. All path parameters are
 
 | Parameter | Data type | Description
 | :--- | :--- | :---
-| `sortString` | String | Determines how to sort the results. Defaults to `monitor_name.keyword`.
-| `sortOrder` | String | Determines the order of the results. Options are `asc` or `desc`. Defaults to `asc`.
-| `missing` | String | Optional.
-| `size` | String | Determines the size of the request to be returned. Defaults to `20`.
-| `startIndex` | String | The index to start from. Used for paginating results. Defaults to `0`.
-| `searchString` | String | A search string used to look for a specific alert. Defaults to an empty string.
-| `severityLevel` | String | The severity level to filter for. Defaults to `ALL`.
-| `alertState` | String | The alert state to filter for. Defaults to `ALL`.
+| `sortString` | String | Defines how to sort the results. Default is `monitor_name.keyword`.
+| `sortOrder` | String | Defines the results order. Options are `asc` or `desc`. Default is `asc`.
+| `missing` | String | Specifies whether to include missing data in the response.
+| `size` | String | Defines the size of the request to be returned. Default is `20`.
+| `startIndex` | String | Defines the index to start from. Used for paginating results. Default is `0`.
+| `searchString` | String | Defines the search string to use for searching a specific alert. Default is an empty string.
+| `severityLevel` | String | Defines the severity level to filter for. Default is `ALL`.
+| `alertState` | String | Defines the alert state to filter for. Default is `ALL`.
 | `monitorId` | String | Filters by monitor ID.
+| `workflowIds` | String | Allows for monitoring the status of multiple chained alerts within a single dashboard. Available in OpenSearch 2.9 or later.
```

Comment:
`Allows for monitoring the status of chained alerts from multiple workflows within a single dashboard. Available in OpenSearch 2.9 or later.`